frontend/lib/requests/authorized-user.ts

```
@@ -346,33 +351,228 @@
+    if (isEnabled !== undefined) data.isEnabled = isEnabled;
+    if (firstTime !== undefined) data.firstTime = updates.firstTime;
+    if (linkedin !== undefined) data.linkedin = linkedin;
+    if (github !== undefined) data.github = github;
+    if (photo !== undefined) data.profilePhoto = photo;
+    if (roles && roles.length > 0) {
+      data.roles = {
+        set: roleIds.map((id) => ({ id })),
+      };
+    }
+
     const response = await fetch(
```

ESLint

[error] 'response' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)